include/boost/geometry/formulas/karney_direct.hpp

```diff
@@ -0,0 +1,290 @@
+// Boost.Geometry
+
+// Contributed and/or modified by Adeel Ahmad.
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_GEOMETRY_FORMULAS_KARNEY_DIRECT_HPP
+#define BOOST_GEOMETRY_FORMULAS_KARNEY_DIRECT_HPP
+
+
+#include <boost/math/constants/constants.hpp>
+#include <boost/math/special_functions/hypot.hpp>
+
+#include <boost/geometry/util/math.hpp>
+#include <boost/geometry/util/series_expansion.hpp>
+
+#include <boost/geometry/formulas/flattening.hpp>
+#include <boost/geometry/formulas/result_direct.hpp>
+
+
+namespace boost { namespace geometry { namespace formula
+{
+
+
+/*!
+\brief The solution of the direct problem of geodesics on latlong coordinates,
+       after Karney (2011).
+\author See
+- Charles F.F Karney, Algorithms for geodesics, 2011
+https://arxiv.org/pdf/1109.4448.pdf
+*/
+template <
+    typename CT,
+    size_t SeriesOrder = 8,
```

Comment:
maybe place it at the end of parameters' list to be more consistent with similar strategies

Reply:
Thanks! Just updated this.